deploy/olm/storageos/storageosoperator.0.0.10.clusterserviceversion.yaml

```diff
@@ -79,6 +79,11 @@ spec:
     2GB of RAM.  Runs in user-space on any 64-bit Linux with no custom kernel
     modules.
 
+    ## Required Parameters
+
+    * `secretRefName` - provide the name of a secret that contains two keys for the `apiUsername` and `apiPassword` of the default account ([documentation](https://docs.storageos.com/docs/reference/cluster-operator/examples))
+    * `secretRefNamespace` - provide the namespace of the above secret
```

Comment:
"provide the namespace of the above secret" -> "the namespace where the api credentials secret is stored"